File: app/src/main/java/com/meiji/daily/module/useradd/UserAddComponent.java
package com.meiji.daily.module.useradd;

import com.meiji.daily.di.component.AppComponent;
import com.meiji.daily.di.scope.FragmentScoped;

import dagger.Component;

/**
 * Created by Meiji on 2017/12/28.
 */
@FragmentScoped
@Component(modules = UserAddModule.class, dependencies = AppComponent.class)
public interface UserAddComponent {

    void inject(UserAddView userAddView);
}

File: app/src/main/java/com/meiji/daily/module/useradd/UserAddComponent.kt
package com.meiji.daily.module.useradd

import com.meiji.daily.di.component.AppComponent
import com.meiji.daily.di.scope.FragmentScoped

import dagger.Component

/**
 * Created by Meiji on 2017/12/28.
 */
@FragmentScoped
@Component(modules = arrayOf(UserAddModule::class), dependencies = arrayOf(AppComponent::class))
interface UserAddComponent {

    fun inject(userAddView: UserAddView)
}
